config/contentDatabaseConfGenerated.json

```diff
@@ -1169,6 +1169,42 @@
       ],
       "field-value-searches": true
     },
+    {
+      "field-name": "workAboutWorkId",
+      "field-path": [
+        {
+          "path": "/json[type = ('VisualItem', 'LinguisticObject')]/about[./type = ('HumanMadeObject', 'DigitalObject')]/id",
+          "weight": 1
+        },
+        {
+          "path": "/json[type='Set'][classified_as/equivalent/id='http://vocab.getty.edu/aat/300375748']/about[./type = ('HumanMadeObject', 'DigitalObject')]/id",
+          "weight": 1
+        },
+        {
+          "path": "/json[type = 'VisualItem']/represents[./type = ('HumanMadeObject', 'DigitalObject')]/id",
+          "weight": 1
+        }
+      ],
+      "field-value-searches": true
+    },
+    {
+      "field-name": "workAboutItemId",
```

Comment:
@kamerynB  the xpaths here for `workAboutItemId` are the ones that should go into `workAboutWorkId` and vice versa

Reply:
These should be fixed now.